extensions/wikia/UserProfilePageV3/UserProfilePageController.class.php

```diff
@@ -958,6 +958,8 @@ public function renderClosingModal() {
 
 	public function removeavatar() {
```

Comment:
just a small thing but you could add the doc with info that this function throw BadRequest Exception